apps/src/applab/AppLabView.jsx

```diff
@@ -1,10 +1,19 @@
 /** @file Top-level view for App Lab */
 'use strict';
 
+var _ = require('lodash');
```

Comment:
Did you mean to require full lodash here?  If you want the apps-built version this needs to be `require('../lodash');` right?